src/main/java/mr/limpios/smart_divide_backend/domain/models/BalanceInfo.java

```diff
@@ -0,0 +1,14 @@
+package mr.limpios.smart_divide_backend.domain.models;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+import java.math.BigDecimal;
+@Data
+@AllArgsConstructor
+public class BalanceInfo {
+    private String name;
+    private BigDecimal balance;
+
+}
+
```

Comment:
Considera si esta clase es necesaria, si lo es, podrías cambiarla por un record o una clase normal. Utilizar anotaciones de lombok en dominio, me parece que rompe la idea de clean architecture

```suggestion
package mr.limpios.smart_divide_backend.domain.models;

import java.math.BigDecimal;
public record BalanceInfo (private String name, BigDecimal balance) {}

```